src/test/java/core/basesyntax/fruitoperation/OperationsTest.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.fruitoperation;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class OperationsTest extends TestCase {
```

Comment:
```suggestion
public class OperationsTest {
```
can we go without this in all tests?

Reply:
done